README.md

```diff
@@ -78,11 +78,19 @@ If you are not running this client on Google Compute Engine, you need a Google D
 // Authenticating on a global basis.
 var projectId = process.env.GCLOUD_PROJECT; // E.g. 'grape-spaceship-123'
 
+// Using a string of your keyfile.json path
 var gcloud = require('gcloud')({
   projectId: projectId,
   keyFilename: '/path/to/keyfile.json'
 });
 
+// Or you can use your keyfile.json directly
```

Comment:
Lose this object if we go with the idea from https://github.com/GoogleCloudPlatform/gcloud-node/pull/1446/files#r71899332